evergreen/config.yml

```diff
@@ -937,6 +937,8 @@ buildvariants:
   - name: fuzzer-tests
     cron: "@daily"
     patchable: false
```

Comment:
I think you def want this to be patchable? i.e. we want to be able to manually run the fuzzer without having to wait for the next day.

Reply:
fixed